specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip/checkNumberRouting": {
+      "get": {
+        "tags": [
+          "NumberRoutingValidation"
+        ],
+        "summary": "Gets the list of routes matching the target phone number, ordered by priority.",
+        "operationId": "CheckNumberRouting",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "targetPhoneNumber",
+            "description": "Phone number to test routing patterns against",
+            "required": true,
+            "type": "string",
+            "x-example": "+11234567890"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "query",
+            "name": "select",
+            "description": "This parameter can be used to select only specific fields of the TrunkRoute objects in the matchingRoutes array.",
+            "required": false,
+            "type": "string",
+            "x-example": "name"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list of matching routes",
+            "schema": {
+              "$ref": "#/definitions/RoutesForNumber"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
```

Comment:
It would be great if you could update the version of Common to the latest stable - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/communication/data-plane/Common/stable/2022-07-13

Reply:
Thanks, it'd be good to update all the references in the file.

Reply:
Updated in the newer commits